Agents/DevInstAgent/README.md

```diff
@@ -0,0 +1,171 @@
+# Device Instantiation Agent
+
+This agent is responsible for instantiating sensors, microcontrollers and devices in a knowledge graph. The agent took in informations from a `.json` file (the device descriptor) and instantiates microcontroller modules with their respective sensor and readings instances. The agent is designed with the device instantiation framework in mind.
+
+Instantiation is based on ontodevice ontology.
+
+## Requirements
+
+### Device Instantiation framework
+The device instantiation framework is a framework to allow fast and consistent sensor instantiation on the knowledge graph to allow faster deployment and development. It can be summarised in the following figure:
+![Device instantiation framework](./readme_img/framework.png)
+
+Sensors will be connected to a microcontroller and it will send data to an IoT database. For this agent, Thingsboard is used for the development of the agent. The Post-Processing agent will pull the data from the database and instantiate a timeseries of the data on the knowledge graph. This agent work with and without the use of the stack. If the stack is used, the agent will send the data to the [data bridge agent](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Agents/DataBridgeAgent) for instantiation. 
+
+The post processing agent will also be responsible for instantiating agent and derivation instances when needed. THe post processing agent is also responsible for creating the timeseries instances. Hence, for different projects, different post processing agnets may be needed. An example of this post-processing agent is the [FHAgent](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Agents/FHAgent) which is responsible for handling fumehood occupancy timeseries data and derivation instances. 
+
+ The device instantiaion agent will be responsible for instantiating the sensors and devices in the knowledge graph using a device descrpitor file written by the user.  
```

Comment:
Spelling error in "descrpitor", similar errors below. I have messaged you the errors via Skype. Please take a look and rectify them.